manifests/init.pp

```diff
@@ -102,9 +105,9 @@
   $use_sympa           = false,         # postfix_use_sympa
   $postfix_ensure      = 'present',
   $mailx_ensure        = 'present',
-) inherits postfix::params {
-
+) { include ::postfix::params
```

Comment:
Have created #167 to cover this.  It will need to be approved first, this change will fail without it.